src/seb/registered_tasks/mteb_tasks_clustering.py

```diff
@@ -0,0 +1,176 @@
+        labels = [labels[i : i + n_per_split] for i in range(0, n_pairs, n_per_split)][:-1]
+
+        ds = datasets.Dataset.from_dict({"sentences": documents, "labels": labels})
+        self.dataset = datasets.DatasetDict({"all": ds})
+
+
+class VGSummarizationClustering(AbsTaskClustering):
+    @property
+    def description(self) -> dict[str, Any]:
+        return {
+            "name": "VGSummarizationClustering",
+            "hf_hub_name": "navjordj/VG_summarization",
+            "description": "Articles and ingresses from VG news articles extracted from Norsk Aviskorpus. Uses articles classes as clusters.",
+            "reference": "https://huggingface.co/datasets/navjordj/VG_summarization",
+            "type": "Clustering",
+            "category": "p2p",
+            "eval_splits": ["test"],  # 18k examples (what size do we want to for this?)
+            "eval_langs": ["nb"],  # maybe nn too?
+            "main_score": "v_measure",
+            "revision": "d4c5a8ba10ae71224752c727094ac4c46947fa29",
+        }
+
+    def load_data(self, **kwargs: dict):  # noqa: ARG002
+        """
+        Load dataset from HuggingFace hub
+        """
+        if self.data_loaded:
+            return
+
+        self.dataset: datasets.DatasetDict = datasets.load_dataset(
+            self.description["hf_hub_name"],
+            revision=self.description.get("revision"),
+        )  # type: ignore
+
+        self.dataset_transform()
+        self.data_loaded = True
+
+    def dataset_transform(self):
+        splits = self.description["eval_splits"]
+
+        documents = []
+        labels = []
+        label_col = "classes"
+
+        ds = {}
+        for split in splits:
+            ds_split = self.dataset[split]
+
+            _label = self.normalize_labels(ds_split[label_col])
+            documents.extend(ds_split["title"])
+            labels.extend(_label)
+
+            documents.extend(ds_split["ingress"])
+            labels.extend(_label)
+
+            documents.extend(ds_split["article"])
+            labels.extend(_label)
+
+            assert len(documents) == len(labels)
+
+            rng = random.Random(1111)  # local only seed
+            pairs = list(zip(documents, labels))
+            rng.shuffle(pairs)
+            documents, labels = list(zip(*pairs))
+
+            documents_batched = list(batched(documents, 512))[:4]
+            labels_batched = list(batched(labels, 512))[:4]
+            # (512x4) resampling changes scores from 12.68, 11.30, 12.65,
+            # (512x5) resampling changes scores from 11.5, 11.8,
+            # (2048x5) resampling changes scores from 11.40, 10.94, 10.95
+            # (5000x5) resampling changes scores from 10.9, 10.48
+            # conclusion: It seems like the more data gives a slightly better estimate, but not by much.
+            # it also seems like the large batches lead to a slightly lower score (I could imagine it is correlated with number of clusters).
```

Comment:
Did some of experimentation on size of clustering targets. 

@Muennighoff I could imagine you also have done some experiments here. I would love to hear what you have to say.

Of course, it is always nice to keep the dataset "lean" to allow folks to quickly experiment with models.

All of these experiments were with the all-mini-lm.

Reply:
See also #94

Reply:
Yeah I'd keep it lean such that it doesn't take >15min to eval with all-mini-lm

Reply:
Perfect that is done now. I have also reduced the size of SwednClustering as well.